articles/connections/database/migrating.md

```diff
@@ -1,37 +1,76 @@
+- Only **Enterprise** subscription plans include the ability to connect to an existing store or database via JavaScript running on Auth0's servers for every authentication request.
 
-Only **Developer**, **Developer Pro**, and **Enterprise** subscription plans include the database migration feature.
-
-Only **Enterprise** subscription plans include the ability to connect to an existing store or database via JavaScript running on Auth0's servers for every authentication request.
-
-[Click here](https://auth0.com/pricing) to learn more about Auth0 pricing plans.
+For more information refer to [Auth0 pricing plans](https://auth0.com/pricing).
 :::
 
-You can read more about database connections and the the options for using several user stores at [Database Identity Providers](/connections/database).
-
 ## The Migration Process
 
 When a user authenticates via a custom database connection marked for import to Auth0, the following process takes place:
 
 ![Migration Diagram](/media/articles/connections/database/migrating-diagram.png)
 
-Auth0 authenticates migrated users against the Auth0 database. If the user has not been migrated, Auth0 executes your custom login script and, upon successfully log in, adds the user to the Auth0 database. Subsequent logins results in the user's credentials retrieved from Auth0, *not* your custom database.
+Auth0 authenticates migrated users against the Auth0 database.
+
+If the user has not been migrated, Auth0 executes your custom login script and, upon successfully log in, adds the user to the Auth0 database.
+
+Subsequent logins result in the user's credentials retrieved from Auth0, **NOT** your custom database.
 
-New users are automatically added to the Auth0 database..
+New users are automatically added to the Auth0 database.
 
 ::: note
 Auth0 can only assist users in the Auth0 database with password reset.
 :::
 
+## Enable Data Validation
+
+There are certain validations that Auth0 can run before adding the user to the Auth0 database:
+
+- The email must be set and unique
+- If the connection requires a username (i.e. the **Requires Username** toggle is enabled at the connection's settings), then one must be set
+- If a username is set, then it must be unique
+- The email format must be valid
+
+Additionally, both the email and the username will be converted to lowercase before they are stored.
+
+You can enable these validations for a connection using the [Update a connection endpoint](/api/management/v2#!/Connections/patch_connections_by_id) of the Management APIv2.
+
+```har
+{
+  "method": "PATCH",
+  "url": "https://${account.namespace}/api/v2/connections/{connection-id}",
+  "httpVersion": "HTTP/1.1",
+  "cookies": [],
+  "headers": [
+    { "name": "Authorization", "value": "Bearer {access-token}" }, 
+    { "name": "Content-Type", "value": "application/json"}
+  ],
+  "queryString": [],
+  "postData": {
+    "mimeType": "application/json",
+    "text": "{ \"options\": { \"strategy_version\": 2 } }"
```

Comment:
can we add something here to make it clear that there are missing attributes in the options object to avoid people overriding if they dont read further?

something like `"text": "{ \"options\": { \"strategy_version\": 2, ... } }"` (not sure how do you use to handle this cases in other docs)

maybe even a comment pointing them to read the note about this.

You know, people will copy paste and run it without reading the following notes